src/sass/_recommendations_page.scss

```diff
@@ -3,23 +3,23 @@ $card-gap: 24px;
 .recommendations-container__card-list {
   gap: $card-gap $card-gap;
```

Comment:
[question]: The $card-gap variable is local to this repository and is not provided by Paragon. Do we want to provide the ability to change it via design tokens/CSS variables? This also applies to other local SCSS variables.

For example:
```
$card-gap: var(--card-gap, 24px);
```

Reply:
In most cases we won't need to edit it, so let's not create a new CSS variable for --card-gap.